2023-06-27 12:23:45,120 INFO    MainThread:530897 [wandb_setup.py:_flush():76] Current SDK version is 0.15.4
2023-06-27 12:23:45,120 INFO    MainThread:530897 [wandb_setup.py:_flush():76] Configure stats pid to 530897
2023-06-27 12:23:45,120 INFO    MainThread:530897 [wandb_setup.py:_flush():76] Loading settings from /home/macula/SMATousi/.config/wandb/settings
2023-06-27 12:23:45,121 INFO    MainThread:530897 [wandb_setup.py:_flush():76] Loading settings from /home/macula/SMATousi/VAE_outlier/outlier_VAE/wandb/settings
2023-06-27 12:23:45,121 INFO    MainThread:530897 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2023-06-27 12:23:45,121 INFO    MainThread:530897 [wandb_setup.py:_flush():76] Applying setup settings: {'_disable_service': False}
2023-06-27 12:23:45,121 INFO    MainThread:530897 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2023-06-27 12:23:45,121 INFO    MainThread:530897 [wandb_init.py:_log_setup():507] Logging user logs to /home/macula/SMATousi/VAE_outlier/outlier_VAE/wandb/run-20230627_122343-nvrnrzfj/logs/debug.log
2023-06-27 12:23:45,121 INFO    MainThread:530897 [wandb_init.py:_log_setup():508] Logging internal logs to /home/macula/SMATousi/VAE_outlier/outlier_VAE/wandb/run-20230627_122343-nvrnrzfj/logs/debug-internal.log
2023-06-27 12:23:45,121 INFO    MainThread:530897 [wandb_init.py:init():547] calling init triggers
2023-06-27 12:23:45,122 INFO    MainThread:530897 [wandb_init.py:init():555] wandb.init called with sweep_config: {}
config: {}
2023-06-27 12:23:45,122 INFO    MainThread:530897 [wandb_init.py:init():572] re-initializing run, found existing run on stack: 83qdtxz7
2023-06-27 12:23:45,127 INFO    MainThread:530897 [wandb_run.py:_finish():1890] finishing run tousi-team/VAE_Outlier_Creditcard_Training_1/83qdtxz7
2023-06-27 12:23:45,128 INFO    MainThread:530897 [jupyter.py:save_history():445] not saving jupyter history
2023-06-27 12:23:45,128 INFO    MainThread:530897 [jupyter.py:save_ipynb():373] not saving jupyter notebook
2023-06-27 12:23:45,128 INFO    MainThread:530897 [wandb_init.py:_jupyter_teardown():435] cleaning up jupyter logic
2023-06-27 12:23:45,128 INFO    MainThread:530897 [wandb_run.py:_atexit_cleanup():2124] got exitcode: 0
2023-06-27 12:23:45,129 INFO    MainThread:530897 [wandb_run.py:_restore():2107] restore
2023-06-27 12:23:45,129 INFO    MainThread:530897 [wandb_run.py:_restore():2113] restore done
2023-06-27 12:23:46,428 INFO    MainThread:530897 [wandb_init.py:init():596] starting backend
2023-06-27 12:23:46,428 INFO    MainThread:530897 [wandb_init.py:init():600] setting up manager
2023-06-27 12:23:46,432 INFO    MainThread:530897 [backend.py:_multiprocessing_setup():108] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2023-06-27 12:23:46,435 INFO    MainThread:530897 [wandb_init.py:init():606] backend started and connected
2023-06-27 12:23:46,440 INFO    MainThread:530897 [wandb_run.py:_label_probe_notebook():1235] probe notebook
2023-06-27 12:23:46,440 INFO    MainThread:530897 [wandb_run.py:_label_probe_notebook():1245] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2023-06-27 12:23:46,440 INFO    MainThread:530897 [wandb_init.py:init():703] updated telemetry
2023-06-27 12:23:47,592 INFO    MainThread:530897 [wandb_init.py:init():736] communicating run to backend with 60.0 second timeout
2023-06-27 12:23:56,108 WARNING MainThread:530897 [wandb_init.py:init():1174] interrupted
Traceback (most recent call last):
  File "/home/macula/SMATousi/.conda/envs/mac-deep/lib/python3.7/site-packages/wandb/sdk/wandb_init.py", line 1152, in init
    run = wi.init()
  File "/home/macula/SMATousi/.conda/envs/mac-deep/lib/python3.7/site-packages/wandb/sdk/wandb_init.py", line 742, in init
    cancel=True,
  File "/home/macula/SMATousi/.conda/envs/mac-deep/lib/python3.7/site-packages/wandb/sdk/lib/mailbox.py", line 283, in wait
    found, abandoned = self._slot._get_and_clear(timeout=wait_timeout)
  File "/home/macula/SMATousi/.conda/envs/mac-deep/lib/python3.7/site-packages/wandb/sdk/lib/mailbox.py", line 130, in _get_and_clear
    if self._wait(timeout=timeout):
  File "/home/macula/SMATousi/.conda/envs/mac-deep/lib/python3.7/site-packages/wandb/sdk/lib/mailbox.py", line 126, in _wait
    return self._event.wait(timeout=timeout)
  File "/home/macula/SMATousi/.conda/envs/mac-deep/lib/python3.7/threading.py", line 552, in wait
    signaled = self._cond.wait(timeout)
  File "/home/macula/SMATousi/.conda/envs/mac-deep/lib/python3.7/threading.py", line 300, in wait
    gotit = waiter.acquire(True, timeout)
KeyboardInterrupt
2023-06-27 12:25:45,349 INFO    MainThread:530897 [wandb_setup.py:_flush():76] Current SDK version is 0.15.4
2023-06-27 12:25:45,350 INFO    MainThread:530897 [wandb_setup.py:_flush():76] Configure stats pid to 530897
2023-06-27 12:25:45,351 INFO    MainThread:530897 [wandb_setup.py:_flush():76] Loading settings from /home/macula/SMATousi/.config/wandb/settings
2023-06-27 12:25:45,351 INFO    MainThread:530897 [wandb_setup.py:_flush():76] Loading settings from /home/macula/SMATousi/VAE_outlier/outlier_VAE/wandb/settings
2023-06-27 12:25:45,351 INFO    MainThread:530897 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2023-06-27 12:25:45,351 INFO    MainThread:530897 [wandb_setup.py:_flush():76] Applying setup settings: {'_disable_service': False}
2023-06-27 12:25:45,351 INFO    MainThread:530897 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2023-06-27 12:25:45,351 INFO    MainThread:530897 [wandb_init.py:_log_setup():507] Logging user logs to /home/macula/SMATousi/VAE_outlier/outlier_VAE/wandb/run-20230627_122543-3e9p5if9/logs/debug.log
2023-06-27 12:25:45,351 INFO    MainThread:530897 [wandb_init.py:_log_setup():508] Logging internal logs to /home/macula/SMATousi/VAE_outlier/outlier_VAE/wandb/run-20230627_122543-3e9p5if9/logs/debug-internal.log
2023-06-27 12:25:45,352 INFO    MainThread:530897 [wandb_init.py:_jupyter_setup():453] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x7f3d98777e50>
2023-06-27 12:25:45,352 INFO    MainThread:530897 [wandb_init.py:init():547] calling init triggers
2023-06-27 12:25:45,352 INFO    MainThread:530897 [wandb_init.py:init():555] wandb.init called with sweep_config: {}
config: {}
2023-06-27 12:25:45,352 INFO    MainThread:530897 [wandb_init.py:init():596] starting backend
2023-06-27 12:25:45,353 INFO    MainThread:530897 [wandb_init.py:init():600] setting up manager
2023-06-27 12:25:45,356 INFO    MainThread:530897 [backend.py:_multiprocessing_setup():108] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2023-06-27 12:25:45,357 INFO    MainThread:530897 [wandb_init.py:init():606] backend started and connected
2023-06-27 12:25:45,360 INFO    MainThread:530897 [wandb_run.py:_label_probe_notebook():1235] probe notebook
2023-06-27 12:25:45,360 INFO    MainThread:530897 [wandb_run.py:_label_probe_notebook():1245] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2023-06-27 12:25:45,360 INFO    MainThread:530897 [wandb_init.py:init():703] updated telemetry
2023-06-27 12:25:45,980 INFO    MainThread:530897 [wandb_init.py:init():736] communicating run to backend with 60.0 second timeout
2023-06-27 12:26:46,078 ERROR   MainThread:530897 [wandb_init.py:init():762] encountered error: Run initialization has timed out after 60.0 sec. 
Please refer to the documentation for additional information: https://docs.wandb.ai/guides/track/tracking-faq#initstarterror-error-communicating-with-wandb-process-
2023-06-27 12:26:46,148 ERROR   MainThread:530897 [wandb_init.py:init():1170] Run initialization has timed out after 60.0 sec. 
Please refer to the documentation for additional information: https://docs.wandb.ai/guides/track/tracking-faq#initstarterror-error-communicating-with-wandb-process-
Traceback (most recent call last):
  File "/home/macula/SMATousi/.conda/envs/mac-deep/lib/python3.7/site-packages/wandb/sdk/wandb_init.py", line 1152, in init
    run = wi.init()
  File "/home/macula/SMATousi/.conda/envs/mac-deep/lib/python3.7/site-packages/wandb/sdk/wandb_init.py", line 768, in init
    raise error
wandb.errors.CommError: Run initialization has timed out after 60.0 sec. 
Please refer to the documentation for additional information: https://docs.wandb.ai/guides/track/tracking-faq#initstarterror-error-communicating-with-wandb-process-
2023-06-27 12:27:19,564 INFO    MainThread:530897 [wandb_setup.py:_flush():76] Current SDK version is 0.15.4
2023-06-27 12:27:19,566 INFO    MainThread:530897 [wandb_setup.py:_flush():76] Configure stats pid to 530897
2023-06-27 12:27:19,566 INFO    MainThread:530897 [wandb_setup.py:_flush():76] Loading settings from /home/macula/SMATousi/.config/wandb/settings
2023-06-27 12:27:19,566 INFO    MainThread:530897 [wandb_setup.py:_flush():76] Loading settings from /home/macula/SMATousi/VAE_outlier/outlier_VAE/wandb/settings
2023-06-27 12:27:19,566 INFO    MainThread:530897 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2023-06-27 12:27:19,567 INFO    MainThread:530897 [wandb_setup.py:_flush():76] Applying setup settings: {'_disable_service': False}
2023-06-27 12:27:19,567 INFO    MainThread:530897 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2023-06-27 12:27:19,567 INFO    MainThread:530897 [wandb_init.py:_log_setup():507] Logging user logs to /home/macula/SMATousi/VAE_outlier/outlier_VAE/wandb/run-20230627_122717-6ds1vzou/logs/debug.log
2023-06-27 12:27:19,567 INFO    MainThread:530897 [wandb_init.py:_log_setup():508] Logging internal logs to /home/macula/SMATousi/VAE_outlier/outlier_VAE/wandb/run-20230627_122717-6ds1vzou/logs/debug-internal.log
2023-06-27 12:27:19,567 INFO    MainThread:530897 [wandb_init.py:init():547] calling init triggers
2023-06-27 12:27:19,568 INFO    MainThread:530897 [wandb_init.py:init():555] wandb.init called with sweep_config: {}
config: {}
2023-06-27 12:27:19,568 INFO    MainThread:530897 [wandb_init.py:init():596] starting backend
2023-06-27 12:27:19,568 INFO    MainThread:530897 [wandb_init.py:init():600] setting up manager
2023-06-27 12:27:19,572 INFO    MainThread:530897 [backend.py:_multiprocessing_setup():108] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2023-06-27 12:27:19,574 INFO    MainThread:530897 [wandb_init.py:init():606] backend started and connected
2023-06-27 12:27:19,595 INFO    MainThread:530897 [wandb_run.py:_label_probe_notebook():1235] probe notebook
2023-06-27 12:27:19,597 INFO    MainThread:530897 [wandb_run.py:_label_probe_notebook():1245] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2023-06-27 12:27:19,597 INFO    MainThread:530897 [wandb_init.py:init():703] updated telemetry
2023-06-27 12:27:20,200 INFO    MainThread:530897 [wandb_init.py:init():736] communicating run to backend with 60.0 second timeout
2023-06-27 12:27:28,298 WARNING MainThread:530897 [wandb_init.py:init():1174] interrupted
Traceback (most recent call last):
  File "/home/macula/SMATousi/.conda/envs/mac-deep/lib/python3.7/site-packages/wandb/sdk/wandb_init.py", line 1152, in init
    run = wi.init()
  File "/home/macula/SMATousi/.conda/envs/mac-deep/lib/python3.7/site-packages/wandb/sdk/wandb_init.py", line 742, in init
    cancel=True,
  File "/home/macula/SMATousi/.conda/envs/mac-deep/lib/python3.7/site-packages/wandb/sdk/lib/mailbox.py", line 283, in wait
    found, abandoned = self._slot._get_and_clear(timeout=wait_timeout)
  File "/home/macula/SMATousi/.conda/envs/mac-deep/lib/python3.7/site-packages/wandb/sdk/lib/mailbox.py", line 130, in _get_and_clear
    if self._wait(timeout=timeout):
  File "/home/macula/SMATousi/.conda/envs/mac-deep/lib/python3.7/site-packages/wandb/sdk/lib/mailbox.py", line 126, in _wait
    return self._event.wait(timeout=timeout)
  File "/home/macula/SMATousi/.conda/envs/mac-deep/lib/python3.7/threading.py", line 552, in wait
    signaled = self._cond.wait(timeout)
  File "/home/macula/SMATousi/.conda/envs/mac-deep/lib/python3.7/threading.py", line 300, in wait
    gotit = waiter.acquire(True, timeout)
KeyboardInterrupt
